ld.so.conf

```diff
@@ -1,3 +1,6 @@
+# libc default configuration
+/usr/local/lib
```

Comment:
> it looks like default line for all debians/ubuntus and probably other distros. Seems kinda *nix generic to me, IMO.

That's not true for all Linux distributions:
* ❎  Arch
* ❎  CentOS
* ✅  Debian
```
docker run -it --rm base/archlinux sh
sh-4.3# cat /etc/ld.so.conf
#
# /etc/ld.so.conf
#

include /etc/ld.so.conf.d/*.conf

# End of file
sh-4.3# ls -l /etc/ld.so.conf.d/
total 0
```
```
docker run -it --rm centos sh
sh-4.2# cat /etc/ld.so.conf
include ld.so.conf.d/*.conf
sh-4.2# ls -l /etc/ld.so.conf.d/
total 0
sh-4.2# cat /etc/redhat-release
CentOS Linux release 7.2.1511 (Core)
```
```
docker run -it --rm debian sh
Unable to find image 'debian:latest' locally
latest: Pulling from library/debian
5040bd298390: Already exists
Digest: sha256:abbe80c8c87b7e1f652fe5e99ff1799cdf9e0878c7009035afe1bccac129cad8
Status: Downloaded newer image for debian:latest
# cat /etc/ld.so.conf
include /etc/ld.so.conf.d/*.conf

# ls -l /etc/ld.so.conf.d/
total 8
-rw-r--r-- 1 root root 44 Mar 20  2016 libc.conf
-rw-r--r-- 1 root root 68 Nov 28 04:22 x86_64-linux-gnu.conf
# cat /etc/ld.so.conf.d/libc.conf
# libc default configuration
/usr/local/lib
```